panon/backend/source.py

```diff
@@ -1,4 +1,5 @@
 import numpy as np
+import soundcard as sc
```

Comment:
I didn't expect everyone would be using PulseAudio. They can use PortAudio as a fallback. So import soundcard may cause problems for them.

Reply:
ah okay, that's why this was within the body.

I did this for performance reasons as imports during runtime. If an item is imported more than twice it doesn't bug out but it does take up CPU.

Feel free to revert the changes to this.

Reply:
>  If an item is imported more than twice it doesn't bug out but it does take up CPU.

Really? If so, you can put the statement in the head, with "try: except:" things.